game/missiongenerator/aircraft/flightgroupspawner.py

```diff
@@ -283,6 +375,12 @@ def dcs_start_type(self) -> DcsStartType:
             return DcsStartType.Warm
         raise ValueError(f"There is no pydcs StartType matching {self.start_type}")
 
+    def _start_type_at_airfield(
+        self,
+        airfield: Airfield,
```

Comment:
It's used here:

>     def _generate_at_airfield(self, name: str, airfield: Airfield) -> FlyingGroup[Any]:
>         # TODO: Delayed runway starts should be converted to air starts for multiplayer.
>         # Runway starts do not work with late activated aircraft in multiplayer. Instead
>         # of spawning on the runway the aircraft will spawn on the taxiway, potentially
>         # somewhere that they don't fit anyway. We should either upgrade these to air
>         # starts or (less likely) downgrade to warm starts to avoid the issue when the
>         # player is generating the mission for multiplayer (which would need a new
>         # option).
>         self.flight.unit_type.dcs_unit_type.load_payloads()
>         return self.mission.flight_group_from_airport(
>             country=self.country,
>             name=name,
>             aircraft_type=self.flight.unit_type.dcs_unit_type,
>             airport=airfield.airport,
>             maintask=None,
>             start_type=self._start_type_at_airfield(airfield),
>             group_size=self.flight.count,
>             parking_slots=None,
>         )